tla-typechecker/src/test/scala/at/forsyte/apalache/tla/typecheck/etc/TestToEtcExprDecls.scala

```diff
@@ -116,7 +116,8 @@ class TestToEtcExprDecls extends AnyFunSuite with ToEtcExprBase with BeforeAndAf
     // The body is wrapped with the application of an operator that has the signature Bool => Bool.
     // This allows us to check that the assumption has Boolean type.
     val application = mkUniqApp(Seq(parser("Bool => Bool")), assumption)
-    val expected = mkUniqLet("__Assume_" + assume.ID, mkUniqAbs(application), terminal)
+    val expected =
+      mkUniqLet("__Assume_" + assume.definedName.getOrElse(assume.ID.toString), mkUniqAbs(application), terminal)
```

Comment:
To match the change in `ToEtcExpr.scala`

```suggestion
      mkUniqLet(assume.name, mkUniqAbs(application), terminal)
```